akka-projection-jdbc/src/test/scala/akka/projection/jdbc/JdbcOffsetStoreSpec.scala

```diff
@@ -196,23 +196,23 @@ abstract class JdbcOffsetStoreSpec(specConfig: JdbcSpecConfig)
   private val clock = new TestClock
 
   private val settings = JdbcSettings(testKit.system)
-  private val offsetStore = new JdbcOffsetStore(system, settings, specConfig.jdbcSessionFactory, clock)
+  private val offsetStore = new JdbcOffsetStore(system, settings, specConfig.jdbcSessionFactory _, clock)
   private val dialectLabel = specConfig.name
 
   override protected def beforeAll(): Unit = {
     // start test container if needed
     // Note, the H2 test don't run in container and are therefore will run must faster
-    specConfig.initContainer
+    specConfig.initContainer()
```

Comment:
Maybe we should invert and make 2.13 the default one since it's more strict. How many times we will not see this and get a broken build? 

The risk of using 2.13 by default, is that we can easily include a converter that is not available in 2.12.